index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kip's portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header-container">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <div class="header-nav-lg">
+            <img src="images/ic_menu.jpg" alt="image">
+        </div>
```

Comment:
- [ ] Please, wrap these `2 divs` inside a `header` tag, this should represent the header section, by this, the div containing the humburger should be replaced with a `nav` tag instead.